### PR TITLE
Improve autotuner

### PR DESCRIPTION
**Pull request by raminsalehi**

### CODE REVIEW

**Comment by raminsalehi** [comment]
the required changes are completed

File: module_autotune/src/auto_tune.xc [hunk 1]
```diff
@@ -12,7 +12,7 @@ int init_velocity_auto_tuner(VelCtrlAutoTuneParam &velocity_auto_tune, MotionCon
     velocity_auto_tune.counter=0;
     velocity_auto_tune.save_counter=0;
     velocity_auto_tune.array_length=1000;
-    for(int i=1; i<=1000; i++) velocity_auto_tune.actual_velocity[i]=0;
+    for(int i=1; i<=/*velocity_auto_tune.array_length*/1000; i++) velocity_auto_tune.actual_velocity[i]=0;
```
Comment (vtretyakov) [inline module_autotune/src/auto_tune.xc]: Interesting solution. What happened to this structure element?

File: module_autotune/src/auto_tune.xc [hunk 2]
```diff
@@ -121,7 +121,7 @@ int velocity_controller_auto_tune(VelCtrlAutoTuneParam &velocity_auto_tune, Moti
         motion_ctrl_config.velocity_ki = ((int)(velocity_auto_tune.ki));
         motion_ctrl_config.velocity_kd = ((int)(velocity_auto_tune.kd));
 
-        for(int i=0; i<=1000; i++) velocity_auto_tune.actual_velocity[i] = 0;
+        for(int i=0; i<=/*velocity_auto_tune.array_length*/1000; i++) velocity_auto_tune.actual_velocity[i] = 0;
```
Comment (vtretyakov) [inline module_autotune/src/auto_tune.xc]: The same here, does this member exist or not?

**Comment by vtretyakov** [comment]
@romu4ld please review if PR #53 will not create future merge conflicts from master to develop.

**Pull request merged**